### DISCUSSION TITLE
how to perform toggle operation

### DISCUSSION THREAD

**Discussion by naseramuk** [chartbrew/chartbrew#236] [Q&A]

**Comment by razvanilin** [comment]
Hi there, I moved this to the **Discussions** because issues are mainly used for bugs, features, etc.

I'm having a bit of a hard time to understand the question. Can you please clarify what you mean exactly by "tabs" and "Click function"? Maybe talking more about your use-case is a good starting point.

Regarding the videos, I have a list of videos that I need to record and hopefully I'll get to them fairly soon 🤞

**Reply by razvanilin** [reply]
Please do not tag contributors in your posts as not all are still involved with the project and these messages might be perceived as spam.

Regarding the tabs, this is not something you can achieve with Chartbrew. You can, however, create different dashboards for each department. So you can create one dashboard for Marketing, another for Sales, for Operations, etc. Each dashboard will have to be configured separately with datasets that pull data from your difference sources. What data sources are you using currently?
